easybuild/easyconfigs/w/Wannier90/Wannier90-2.0.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'Wannier90'
+version = '2.0.1'
+
+homepage = 'http://www.wannier.org'
+description = """A tool for obtaining maximally-localised Wannier functions"""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.wannier.org/code']
+
+# ugly hack
+prebuildopts = 'rm make.sys; touch make.sys; '
+prebuildopts += 'F90=ifort COMMS=mpi MPIF90=mpiifort FCOPTS="-O2" LDOPTS="-O2" LIBDIR=$LAPACK_LIB_DIR LIBS=$LIBLAPACK '
```

Comment:
why is this more extensive than the one included in the other easyconfig?

also, take into account that `$LIBLAPACK` likely contains spaces, so you need `LIBS="$LIBLAPACK`

in addition, please consider not hardcoding `-O2`, nad using `$FFLAGS` instead with `FCOPTS="$FFLAGS"`; I guess you can also use `$FFLAGS` for `LDOPTS`